modules/i18n/src/models/namespaces/console-ns.ts

```diff
@@ -3218,6 +3218,69 @@ export interface ConsoleNS {
                     darkMode: string;
                 };
             };
+            invite?: {
```

Comment:
Shall we not use optional here?. Since other languages implementing this interface will not be forced to add the translation.